script.js

```diff
@@ -0,0 +1,110 @@
+const menu = document.querySelector('#menu');
+const mobileNav = document.querySelector('.mobile-nav');
+const menulist = document.querySelectorAll('#nav li a');
+
+function toggleMenu() {
+  if (mobileNav.style.display === 'none') {
+    mobileNav.style.display = 'block';
+    menu.src = 'images/close-icon.png';
+  } else {
+    mobileNav.style.display = 'none';
+    menu.src = 'images/menu-icon.png';
+  }
+}
+
+menu.addEventListener('click', toggleMenu);
```

Comment:
[OPTIONAL] The user has to click twice on the hamburger menu to open it. I will recommend fixing that to improve the user experience.

Reply:
I'll work on the logic, thank you :)